README.adoc

```diff
@@ -13,6 +13,8 @@ To build the samples do:
 
 * `dynamic-source` publishes messages to dynamically created destinations.
 
+* `kinesis-produce-consume` An example application using spring-cloud-stream-binder-aws-kinesis. Presents a web endpoint to send Orders, these are placed on a kinesis stream and then consumed by the application from that stream. 
```

Comment:
Kinesis is a product - should be capitalized.
The `spring-cloud-stream-binder-aws-kinesis` is an artifact - should be as a code in formatting.